subsystems/DrivetrainBase.java

```diff
@@ -1,5 +1,65 @@
 package org.frc5587.lib.subsystems;
 
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+
 public class DrivetrainBase {
-    
-}
+    // create leader and follower motors for the drivetrain
+    private final WPI_TalonFX leftLeader = new WPI_TalonFX(0);
```

Comment:
In these classes, none of the variables should be private or final - they should all be protected so they can get overwritten later if necessary.